INSTALL.txt

```diff
@@ -168,9 +190,8 @@ MS compilers
 If you are familiar with MS tools, that's obviously the easiest path, and
 the compilers are hopefully more mature (although in my experience, they
 are quite fragile, and often segfault on invalid C code). The main drawback
-is that no fortran compiler + MS compiler combination has been tested -
-mingw-w64 gfortran + MS compiler does not work at all (it is unclear
-whether it ever will).
+is that mingw-w64 gfortran + MSVC does not work at all (it is unclear
```

Comment:
Ah okay, I see what you meant. That makes sense. I wonder whether we should completely remove that sentence then - I expect mingw-w64 upstream will be improving over time, and have no idea what the current status is.